src/oscar/static_src/oscar/scss/page/type.scss

```diff
@@ -0,0 +1,97 @@
+// Re setting margins to the bottom only
+h1,
+h2,
+h3,
+h4,
+h5,
+h6,
+p,
+form,
+.control-group,
+.table,
+.page-header,
+.sub-header {
+  margin: 0 0 $base-margin-bottom;
+}
+
+ul,
+ol {
+  margin-bottom: $base-margin-bottom;
+}
+
+.page-header,
+.sub-header,
+.table {
+  h1, h2, h3, h4, h5, h6, form {
+    margin-bottom: 0;
+  }
+}
+
+// Heading fonts a little smaller
+h1, .h1 {
```

Comment:
We should be modifying the sass variables provided by Bootstrap (`$h1-font-size` etc.), rather than adding additional styles here.